client/app/domain/domain.service.js

```diff
@@ -1314,6 +1314,22 @@ angular.module("services").service(
         }
 
         /**
+         * Get DNS Anycast status
+         * @param {string} serviceName
+         */
+        getDnsAnycast (serviceName) {
+            return this.OvhHttp
+                .get(`/domain/zone/${serviceName}`, { rootPath: "apiv6" })
+                .then((response) => {
+                    if (response.hasDnsAnycast != null) {
```

Comment:
Some people don't like `!= null`, you should use instead`_(response.hasDnsAnycast).isBoolean()`